lib/vagrant-openshift/action/install_openshift_base_dependencies.rb

```diff
@@ -109,6 +109,7 @@ def call(env)
 fi
 
 GOPATH=/data go get golang.org/x/tools/cmd/vet
+cd /data/src/golang.org/x/tools && git checkout c262de870b618eed648983aa994b03bc04641c72 && cd -
```

Comment:
Probably a comment as to why this is needed would be good as well. Is this particular version a "known good" version?